dev/breeze/SELECTIVE_CHECKS.md

```diff
@@ -77,7 +78,8 @@ The logic implements the following rules:
 * `Full tests` mode is enabled when the event is PUSH, or SCHEDULE or when "full tests needed" label is set.
   That enables all matrix combinations of variables, and all possible tests
 * Python, Kubernetes, Backend, Kind, Helm versions are limited to "defaults" only unless `Full tests` mode
-  is enabled.
+  is enabled. Tests for Snowflake are always performed on Python 3.8 as well, as `snowflake-snowpark-python`
```

Comment:
I think (similarly as above) - this is wrong. If snowpark does not work on other Python than 3.8 then we should not add it as community provider. We have currently NO providers that work only on specific version of Python. Much better solution is if snowflake relases their own "snowpark" provider - only available on 3.8 and limiting beam and cloudpickle rather than trying to make it part of the snowflake provider.  It COULD be an optional feature of the snowflake provider (with some way of detecting when it is used outside of it) but then it should not really impact limiting other providers that use the same dependencies I think.